src/targets/gpu/target.cpp

```diff
@@ -103,6 +103,10 @@ std::vector<pass> target::get_passes(migraphx::context& gctx, const compile_opti
         unsupported_fp8_ops.insert("dot");
         unsupported_fp8_ops.insert("quant_dot");
     }
+#else
+    // mlir doesn't support fp8 dot
+    unsupported_fp8_ops.insert("dot");
+    unsupported_fp8_ops.insert("quant_dot");
```

Comment:
This just duplicating code for both branches. It would be better to have `rocblas_fp8_available` function that just always returns false when rocblas is disabled.